src/generators/minecraft-character-allay/MinecraftAllayCharacterGenerator.res

```diff
@@ -0,0 +1,1154 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "minecraft-allay-character"
+
+let name = "Minecraft Allay Character"
+
+let images: array<Generator.imageDef> = [{id: "Overlay", url: requireImage("OverlayAllay.png")}]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin1",
+    url: requireTexture("Skin1.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Skin2",
+    url: requireTexture("Skin2.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+]
+
+module TextureMap = {
+  type cuboid = {
+    top: Builder.rectangle,
+    bottom: Builder.rectangle,
+    front: Builder.rectangle,
+    right: Builder.rectangle,
+    left: Builder.rectangle,
+    back: Builder.rectangle,
+    topoverlay: Builder.rectangle,
+    bottomoverlay: Builder.rectangle,
+    frontoverlay: Builder.rectangle,
+    rightoverlay: Builder.rectangle,
+    leftoverlay: Builder.rectangle,
+    backoverlay: Builder.rectangle,
+  }
+
+  type t = {
+    head: cuboid,
+    body: cuboid,
+    rleg: cuboid,
+    lleg: cuboid,
+    rarm: cuboid,
+    larm: cuboid,
+    rarmAlex: cuboid,
+    larmAlex: cuboid,

```

Comment:
Let's also rename these variables for clarity and also use camelCase (you should be able to do a search and replace to do it quickly).

```res
type t = {
  head: cuboid,
  body: cuboid,
  rightLeg: cuboid,
  leftLeg: cuboid,
  rightArm: cuboid,
  leftArm: cuboid,
  rightArmAlex: cuboid,
  leftArmAlex: cuboid,
}
```